src/main/java/one/util/streamex/EntryStream.java

```diff
@@ -2284,4 +2285,88 @@ public void forKeyValue(BiConsumer<? super K, ? super V> action) {
                 Stream.generate(() -> new SimpleImmutableEntry<>(keySupplier.get(), valueSupplier.get())),
                 StreamContext.SEQUENTIAL);
     }
+
+    /**
+     * Returns a {@code EntryStream} consisting of the elements of this stream those keys are not equal to given keys.
+     *
+     * <p>
+     * This is an <a href="package-summary.html#StreamOps">intermediate</a> operation.
+     *
+     * @return the new stream containing only the values
+     * @see #withoutKey(Object)
+     * @see #withoutValue(Object)
+     * @see #withoutValues(Object...)
+     * @since 0.7.4
+     */
+    public EntryStream<K, V> withoutKeys(K... keys) {
+        if (keys.length == 0)
+            return this;
+        if (keys.length == 1)
+            return withoutKey(keys[0]);
+        return filter(entry -> {
+            for (K key : keys) {
+                if (entry.getKey().equals(key))
+                    return false;
+            }
+            return true;
+        });
+    }
+
+    /**
+     * Returns a {@code EntryStream} consisting of the elements of this stream those keys are not equal given key.
+     *
+     * <p>
+     * This is an <a href="package-summary.html#StreamOps">intermediate</a> operation.
+     *
+     * @return the new stream containing only the values
+     * @see #withoutKeys(Object...)
+     * @see #withoutValue(Object)
+     * @see #withoutValues(Object...)
+     * @since 0.7.4
+     */
+    public EntryStream<K, V> withoutKey(K key) {

```

Comment:
Let's skip non-vararg versions for now. As they named differently, they may pollute API adding the confusion. Just inline them into `withoutKeys` and `withoutValues`.